awscli/examples/schemas/get-discovered-schema.rst

```diff
@@ -0,0 +1,15 @@
+**To retrieve the discovered schema that was created by the Schemas service based on a sample event.**
+
+The following ``get-discovered-schema`` retrieves the discovered schema that was created by the Schemas service based on a sample event. ::
+
+	aws schemas get-discovered-schema \
+		--events "{\"Source\":\"com.mycompany.myapp\",\"Detail\":\"{ \\\"key1\\\": \\\"value1\\\", \\\"key2\\\": \\\"value2\\\" }\",\"EventBusName\":\"default\",\"Resources\":[\"resource1\",\"resource2\"],\"DetailType\":\"myDetailType\"}" \
```

Comment:
Quotation rules for command examples must use unix-like formatting rules. This means using single quotes and unescaping your double quotes similar to the following:

```suggestion
		--events '{"Source":"com.mycompany.myapp","Detail":"{ \\"key1\\": \\"value1\\", \\"key2\\": \\"value2\\" }","EventBusName":"default","Resources":["resource1","resource2"],"DetailType":"myDetailType"}' \
```

Please update all examples not following these rules.
https://aws.github.io/aws-cli/docs_styleguide.html#command

Reply:
updated